R/spec.R

```diff
@@ -73,7 +73,7 @@ spec_fn <- function(call, ctxt) {
   }
 
   # Deparse so non-syntactic names are backticked
-  expr_deparse(fn)
+  deparse(fn)
```

Comment:
Probably want to use some homegrown equivalent to `deparse1` to ensure we only get 1 string back? `paste(deparse(fn), collapse = "")` would probably be ok